airflow/www/utils.py

```diff
@@ -717,10 +717,10 @@ def clean_column_names():
 
         clean_column_names()
         # Support for AssociationProxy in search and list columns
-        for desc in self.obj.__mapper__.all_orm_descriptors:
+        for obj_attr, desc in self.obj.__mapper__.all_orm_descriptors.items():
             if not isinstance(desc, AssociationProxy):
                 continue
-            proxy_instance = getattr(self.obj, desc.value_attr)
+            proxy_instance = getattr(self.obj, obj_attr)
```

Comment:
previously the code assumed that the attr name where the proxy is stored on the object (`obj_attr`) is the same as the name of the field on the remote object (`desc.value_attr`).  but it doesn't have to be the case.  and here we store the reference to `task_instance_note.content` under the attr `notes`